src/folders.js

```diff
@@ -0,0 +1,34 @@
+const client = require('./lib/client')
+const log = require('./lib/logger')
+
+const operations = {
+  get: async (folderId) => {
+    const folder = await client.folders.get(folderId)
+    log(folder)
+    return folder
+  },
+  create: async (name, { parentID }) => {
+    const folder = await client.folders.create(parentID, name)
+    log(folder)
```

Comment:
Minor suggestion: The CLI option in `index.js` is `parentId`. For consistency, consider renaming `parentID` to `parentId` here in the function signature and when accessing it from the `options` object.